Fix generating test examples

The make target for `examples` references as script that does not exist (`bin/generate-examples.php`)

```
$ make examples 
./vendor/bin/hh-autoload
/Users/greenberg/code/hack-json-schema/vendor/autoload.hack
hhvm ./bin/generate-examples.php

Notice: File could not be loaded: ./bin/generate-examples.php
make: *** [examples] Error 255
```

--- a/Makefile
+++ b/Makefile
@@ -25,4 +25,4 @@ format:
 	find {src,tests} -type f -exec hackfmt -i {} \;
 
 examples: hh_autoload
-	hhvm ./bin/generate-examples.php
+	./vendor/bin/hacktest tests

